ansible/deploy_scheduler_runner.yml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: scheduler_runner
+  remote_user: root
+  name: deploy scripts scheduler
+  vars:
+    config_file: /root/.config/freeipa-pr-ci/config.yml
+  tasks:
+    - cron:
```

Comment:
I've assumed that we would not spawn a VM to only use it as a tool to open and close PRs. I've assumed that we would deploy this tool in some PR CI runner. That's why I'm not installing packages and assuming that `config.yml` is already there. I think that this is the way to go because it uses our resources in a better way.